src/app/Http/Controllers/CrudController.php

```diff
@@ -137,10 +137,33 @@ public function edit($id)
     {
         $this->crud->hasAccessOrFail('update');
 
+        // Get initial list of save actions.
```

Comment:
Why didn't all the below go into the `getSaveAction()` method? Probably because it doesn't belong there.

That said, I wonder if the below code should be lifted out.

Reply:
Mainly because `getSaveAction()` is in a trait, so I didn't want to add further responsibility to it, but the code could be moved to it, or refactored in general.
I can easily rewrite it and place it in the trait, which will probably be leaner and cleaner than this.

Reply:
Leaner and cleaner is better than...not leaner and cleaner.